src/client/datascience/kernel-launcher/remoteKernelFinder.ts

```diff
@@ -165,13 +166,15 @@ export class RemoteKernelFinder implements IRemoteKernelFinder {
                     )}`
                 );
 
+                sendKernelListTelemetry(resource, items);
```

Comment:
Now we don't need to wait for users to select a kernel, else we only have part of the data.